libbpf-tools/biolatency.bpf.c

```diff
@@ -59,6 +67,11 @@ int trace_rq_start(struct request *rq, int issue)
 SEC("tp_btf/block_rq_insert")
 int block_rq_insert(u64 *ctx)
 {
+	if(cgroup)
+	{
+		if (!bpf_current_task_under_cgroup(&cgroup_map, 0))
+			return 0;
+	}
```

Comment:
combine these two conditions with `&&` ?

Reply:
Please also follow the kernel code style. Add a space after `if` and drop the brackets.